firefox-ios/Client/FeatureFlags/NimbusFlaggableFeature.swift

```diff
@@ -38,6 +38,15 @@ enum NimbusFeatureFlagID: String, CaseIterable {
     case toolbarRefactor
     case trackingProtectionRefactor
     case zoomFeature
+
+    var debugKey: String? {
+        switch self {
```

Comment:
If everything can have a debug setting, technically, should we just omit the switch and just `return rawValue + PrefsKeys.FeatureFlags.DebugSuffixKey` for everything?

Reply:
it can, but I was originally thinking that settings being added to the debug setting should be intentional, so I took this route that devs should add cases, where they're more familiar with the feature and testing with debug setting.

Reply:
totally fair.